lib/themes/dosomething/paraneue_dosomething/templates/campaign/node--campaign.tpl.php

```diff
@@ -451,9 +454,6 @@
             <?php if ($register_voters) : ?>
               <p><?php print $voter_reg_share_modal_text_2 ?></p>
             <?php endif; ?>
-            <div class="padded-gray-box">
-              <?php print $custom_social_share_link ?>
```

Comment:
@chloealee great question -- i think there's a chance we might want that in the future, but for New Year, New Us, it's not something we're going to be needing. Does that help?